### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by SirriRyisa**
Add index.html
Add style.css
Add linters check

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (alonsofalconi) [inline index.html:1]: - [ ] You have done an excellent job with your project and it's looking great :fire: but one of the requirements it's to add a descriptive README and it's recommended to use [this README template](https://github.com/microverseinc/readme-template) and customize it for your project :wink:
image

**Review by alonsofalconi — CHANGES_REQUESTED** [review]
Hi @SirriRyisa,

You have done a great job so far! :muscle:
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## To highlight :dart:
- Linters working properly :heavy_check_mark:
- Well documented PR :heavy_check_mark:
- Your design looks great! :fire:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alonsofalconi — APPROVED** [review]
Hi @SirriRyisa,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SirriRyisa** [comment]
Yeeeee!!!! 🥳🥳🥳 Thank you alonsofl. I will merge the project now.

**Review by rustam-tolipov — COMMENTED** [review]
Hi @SirriRyisa,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Your code is not ready for review. Please make sure that you have a fresh Pull Request that will show the changes required in a specific project. 
You can either create a brand new repository or just reverting previous changes and opening a new PR. 
We need to see piece of code so that we can perform fresh review once again on the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**